test/tests/functional/pbs_qmgr.py

```diff
@@ -124,3 +125,30 @@ def test_listcmd_whitespaces(self):
             if vndef_file is not None:
                 self.mom.delete_vnodes()
                 self.server.manager(MGR_CMD_DELETE, VNODE, id=nodename)
+
+    def test_multi_attributes(self):
+        """
+        Test to verify that if multiple attributes are set
+        simultaneously and out of which one fail then none
+        will be set.
+        """
+
+        a = {'queue_type': 'e', 'enabled': 't', 'started': 't'}
+        self.server.manager(MGR_CMD_CREATE, QUEUE, a, id='workq2')
+
+        a = {'partition': 'foo'}
+        self.server.manager(MGR_CMD_SET, QUEUE, a, id='workq')
+
+        a = {'partition': 'bar'}
+        self.server.manager(MGR_CMD_SET, QUEUE, a, id='workq2')
+
+        a = {'queue': 'workq', 'partition': 'bar'}
+        try:
+            self.server.manager(MGR_CMD_SET, NODE, a,
+                                id=self.mom.shortname)
+        except PbsManagerError as e:
+            msg = "Partition (null) is not part of "
+            msg += "queue for node"
+            self.logger.info("looking for error, %s" % msg)
+            self.assertTrue(_msg in e.msg[0])
```

Comment:
You set a variable named 'msg' and here you're checking \_msg.  I don't see \_msg set anywhere.  Did you mean msg without the underscore?